api/net/Areas/Admin/Controllers/IngestController.cs

```diff
@@ -165,6 +165,29 @@ public async Task<IActionResult> UpdateIngestEnabledStatusAsync(int id, bool new
         return new JsonResult(new IngestModel(model, _serializerOptions));
     }
 
+    /// <summary>
+    /// Update ingest with the specified 'id'.
+    /// </summary>
+    /// <param name="id"></param>
+    /// <returns></returns>
+    [HttpPut("{id}/reset")]
+    [Produces(MediaTypeNames.Application.Json)]
+    [ProducesResponseType(typeof(IngestModel), (int)HttpStatusCode.OK)]
+    [ProducesResponseType(typeof(ErrorResponseModel), (int)HttpStatusCode.BadRequest)]
+    [SwaggerOperation(Tags = new[] { "Ingest" })]
+    public async Task<IActionResult> UpdateIngestResetAsync(int id)
+    {
+        var model = _service.FindById(id) ?? throw new NoContentException();
+        model.State = new Entities.IngestState(model)
```

Comment:
you will blow away the existing `State` if you do this.  Might be better to check if the ingest has a non null `State`, and if so, just update the `State.FailedAttempts = 0`, else new up a `State` like you are doing already